.gitignore

```diff
@@ -1 +1,2 @@
 bin/
+submodules/
```

Comment:
we should manage submodules/ in git so that git submodule is working fine.

Reply:
thanks for spotting this.